src/get.rs

```diff
@@ -0,0 +1,72 @@
+use crate::config::get_config_toml;
+use crate::error::PasswordStoreError;
+use clap::Values;
+use keyring::Keyring;
+use nitrokey::GetPasswordSafe;
+use nitrokey::{connect, SLOT_COUNT};
+use rpassword::prompt_password_stdout;
+use std::io::Write;
+use std::process::{Command, Stdio};
+use std::thread::sleep;
+use std::time::Duration;
+
+fn get_default_username(service: &str) -> Result<String, PasswordStoreError> {
+    let toml = get_config_toml()?;
+    match toml.get(service) {
+        Some(username) => Ok(username.as_str().unwrap().into()),
+        None => Err(PasswordStoreError::NoDefaultUser(
+            format!("No default set for {}", service).into(),
+        )),
+    }
+}
+
+fn xclip(payload: &str) -> Result<(), PasswordStoreError> {
```

Comment:
It's probably a better idea to use [clipboard](https://crates.io/crates/clipboard).